package.json

```diff
@@ -11,58 +11,58 @@
     "build:analyze": "cross-env ANALYZE=true yarn build",
```

Comment:
How do you update all project dependencies?
`npm update` or `yarn upgrade` ?

Reply:
`ncu -u && yarn`
I have [npm-check-updates](https://www.npmjs.com/package/npm-check-updates) installed

Reply:
Didn't know about this, thanks 😄